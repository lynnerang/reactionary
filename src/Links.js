import React from 'react';
import './Links.scss';

function Links(props) {
  return (
    <>
      <h2>Links</h2>
      <h3 className='resources-heading'>React Docs</h3>
      <p>React has great documentation online with up-to-date information on everything within its framework!</p>
      <a className='resources-link' href='https://reactjs.org/docs/getting-started.html'>https://reactjs.org/docs/getting-started.html</a>
      <h3 className='resources-heading'>React Status</h3>
      <p>A weekly roundup of the latest React and React Native links and tutorials.</p>
      <a className='resources-link' href='https://react.statuscode.com/'>https://react.statuscode.com/</a>
      <h3 className='resources-heading'>Other helpful links</h3>
      <a className='resources-link' href='https://www.udemy.com/react-redux/'>https://www.udemy.com/react-redux/</a>
      <a className='resources-link' href='https://reactjs.org/docs/introducing-jsx.html'>https://reactjs.org/docs/introducing-jsx.html</a>
      <a className='resources-link' href='https://css-tricks.com/learning-react-redux/'>https://css-tricks.com/learning-react-redux/</a>
    </>
  )
}

export default Links;