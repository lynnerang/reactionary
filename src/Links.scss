@import './variables.scss';

.resources-link {
  margin: 20px 0px;
  color: $secondary-color;
  display: block;
  
  &:hover {
    color: $blue;
  }
}

.resources-heading {
  margin-top: 30px;
  margin-bottom: 20px;
}