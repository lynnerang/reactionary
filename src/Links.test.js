import React from 'react';
import ReactDOM from 'react-dom';
import Links from './Links.js';
import { shallow } from 'enzyme';


describe('Links', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow (
      <Links 
      />
      )
  });

  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Links 
    />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('Should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

});