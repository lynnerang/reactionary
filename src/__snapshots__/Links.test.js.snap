// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Links Should match the snapshot 1`] = `
<Fragment>
  <h2>
    Links
  </h2>
  <h3
    className="resources-heading"
  >
    React Docs
  </h3>
  <p>
    React has great documentation online with up-to-date information on everything within its framework!
  </p>
  <a
    className="resources-link"
    href="https://reactjs.org/docs/getting-started.html"
  >
    https://reactjs.org/docs/getting-started.html
  </a>
</Fragment>
`;
